.github/workflows/main-checks.yml

```diff
@@ -182,12 +127,14 @@ jobs:
           cd packages/yew-router
           wasm-pack test --chrome --firefox --headless
 
+
   unit_tests:
     name: Unit Tests on ${{ matrix.toolchain }}
     runs-on: ubuntu-latest
     strategy:
       matrix:
         toolchain:
+          # anyway to dynamically grep the MSRV from Cargo.toml?
```

Comment:
You could find out how to do it from here https://github.com/yewstack/yew/blob/master/.github/workflows/benchmark.yml#L105-L116